wasm/jit/jit_arm64.go

```diff
@@ -200,16 +204,41 @@ func (c *arm64Compiler) applyTwoRegistersToRegisterInstruction(instruction obj.A
 	c.addInstruction(inst)
 }
 
+// applyTwoRegistersToNoneInstruction adds an instruction which takes two source operands on registers.
+func (c *arm64Compiler) applyTwoRegistersToNoneInstruction(instruction obj.As, src1, src2 int16) {
+	inst := c.newProg()
+	inst.As = instruction
+	inst.To.Type = obj.TYPE_NONE
+	inst.From.Type = obj.TYPE_REG
+	inst.From.Reg = src1
+	inst.Reg = src2
+	c.addInstruction(inst)
+}
+
 func (c *arm64Compiler) String() (ret string) { return }
 
+func (c *arm64Compiler) pushFunctionParams() {
+	if c.f != nil && c.f.FunctionType != nil {
```

Comment:
nit I prefer early return on things like this as reduces the indent level

```suggestion
	if c.f == nil || c.f.FunctionType == nil {
		return
	}
```